pages/commenting.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { useQuery } from "@tanstack/react-query";
+import { inter } from "../styles/fonts";
+
+const getComments = async () => {
+  const COMMENTS_URL = "https://jsonplaceholder.typicode.com/comments";
+
+  const res = await fetch(`${COMMENTS_URL}`);
+  return res.json();
+};
+
+function Commenting() {
+  type Comment = {
+    id: number;
+    body: string;
+    name: string;
+    email: string;
+  };
+  const { data, error, isLoading } = useQuery({
+    queryKey: ["comments"],
```

Comment:
You should have all your keys in a central place

Reply:
I dont understand this